src/ArrangerConduit.sol

```diff
@@ -74,6 +76,11 @@ contract ArrangerConduit is UpgradeableProxied, IArrangerConduit {
         emit File(what, data);
     }
 
+    function setBroker(address broker, address asset, bool valid) external auth {
+        isBroker[broker][asset] = valid;
+        emit SetBroker(broker, asset, valid);
+    }
+
     /**********************************************************************************************/
     /*** Router Functions                                                                       ***/
```

Comment:
Not related, but I would rename this to `Operator Functions` because we don't use a router anymore.

Reply:
Updated